app/views/trends/index.html.erb

```diff
@@ -35,5 +35,5 @@
 </table>
 <%= will_paginate_with_info @trends %>
 <% unless TrendCounter.unconfigured? %>
-  <%= _("Last updated %s ago") % (time_ago_in_words TrendCounter.order(:created_at).last.created_at) %>
+  <%= (_("Last updated %s") % (date_time_relative(TrendCounter.order(:created_at).last.created_at)) ).html_safe %>
```

Comment:
Composing strings is generally a bad idea in translations because different languages might have different grammar.